spec/services/hyrax/collections/nested_collection_query_service_spec.rb

```diff
@@ -122,7 +81,7 @@
           it 'returns an array of valid collections of the same collection type' do
             expect(scope).to receive(:can?).with(:deposit, coll_c).and_return(true)
             expect(described_class).to receive(:query_solr).with(collection: coll_c, access: :read, scope: scope, limit_to_id: nil, nest_direction: :as_child).and_call_original
-            expect(subject.map(&:id)).to contain_exactly(another.id, coll_e.id)
+            expect(subject.map(&:id)).to contain_exactly(another.id)
```

Comment:
I think this test was incorrect before.  If A -> B -> C -> D -> E then I wouldn't expect E to be valid as an "available child" for C.

Reply:
Test was restored and support readded with https://github.com/samvera/hyrax/pull/5850/commits/dfc2f0963dbb45d77998954b761a15103969c384